pkg/kube/client.go

```diff
@@ -132,6 +138,134 @@ func (c *Client) Create(resources ResourceList) (*Result, error) {
 	return &Result{Created: resources}, nil
 }
 
+func transformRequests(req *rest.Request) {
+	tableParam := strings.Join([]string{
+		fmt.Sprintf("application/json;as=Table;v=%s;g=%s", metav1.SchemeGroupVersion.Version, metav1.GroupName),
+		fmt.Sprintf("application/json;as=Table;v=%s;g=%s", metav1beta1.SchemeGroupVersion.Version, metav1beta1.GroupName),
+		"application/json",
+	}, ",")
+	req.SetHeader("Accept", tableParam)
+
+	// if sorting, ensure we receive the full object in order to introspect its fields via jsonpath
+	req.Param("includeObject", "Object")
+}
+
+func (c *Client) Get(resources ResourceList, reader io.Reader) (string, error) {
+	buf := new(bytes.Buffer)
+	printFlags := get.NewHumanPrintFlags()
+	typePrinter, _ := printFlags.ToPrinter("")
+	printer := &get.TablePrinter{Delegate: typePrinter}
+	objs := make(map[string][]runtime.Object)
+
+	podSelectors := []map[string]string{}
+	err := resources.Visit(func(info *resource.Info, err error) error {
+		if err != nil {
+			return err
+		}
+
+		gvk := info.ResourceMapping().GroupVersionKind
+		vk := gvk.Version + "/" + gvk.Kind
+		obj, err := getResource(info)
+		if err != nil {
+			fmt.Fprintf(buf, "Get resource %s failed, err:%v\n", info.Name, err)
+		} else {
+			objs[vk] = append(objs[vk], obj)
+
+			objs, err = c.getSelectRelationPod(info, objs, &podSelectors)
+			if err != nil {
+				c.Log("Warning: get the relation pod is failed, err:%s", err.Error())
+			}
+		}
+
+		return nil
+	})
+	if err != nil {
+		return "", err
+	}
+
+	var keys []string
+	for key := range objs {
+		keys = append(keys, key)
+	}
+
+	for _, t := range keys {
```

Comment:
When I try to output using yaml or json the `resources` section is one long string. This is because `Get` returns a string rather than a data structure that can be transformed into yaml, json, or a string output. See...
![Screenshot from 2022-09-08 13-39-37](https://user-images.githubusercontent.com/62991/189190245-c7199ff4-78bb-42d6-80cc-8b9ddef70085.png)

`Get` should return a data structure for the resources. Then when they are printed they can be turned into a string, json, or yaml.

The conversion to a string should happen elsewhere and should not be in the client API.

Reply:
@mattfarina thank you for your guidance on this! I believe I was able to make the required changes and successfully test on local with all 3 formats
![image](https://user-images.githubusercontent.com/19557867/189441378-b0eb91d1-a932-4615-984f-f579c78d3eb0.png)
Yaml
![image](https://user-images.githubusercontent.com/19557867/189441656-cc847fc9-0363-4b5b-aa67-86d86cd8047a.png)
Json
![image](https://user-images.githubusercontent.com/19557867/189441776-a41452f5-96da-4919-8fce-ebe416c38d4b.png)

Can you please take a look to see if it works?

Reply:
@mattfarina sorry for repeated pings like I mentioned before we wanted this feature to get merged into helm 3.10 release. Hence was wondering if you take a look at latest changes?